environment.yml

```diff
@@ -93,10 +92,6 @@ dependencies:
   # Our build of this is version 0.0.1 (!!) and we are using parts
   # of the API that are now gone. Need a serious PR to fix this.
 
-  - rmg::rdkit >=2020.03.3.0
```

Comment:
Ooh, if we can switch RDKit to conda-forge that'd be great. And that being the problem would totally not surprise me.  Brings back some PTSD from before conda...

Reply:
![image](https://github.com/ReactionMechanismGenerator/RMG-Py/assets/33505528/e83a151b-a7f8-4923-92c8-04503a9b712a)